lms/static/js/views/message_banner.js

```diff
@@ -37,4 +37,4 @@
 
         return MessageBannerView;
     });
-}).call(this, define || RequireJS.define);
+}).call(this, define || RequireJS.define);
```

Comment:
I have reverted those changes since some other changes were made to `message_banner`. This change only added a different template but the original template has since been enhanced to support icons.